src/common/types/store-data/assessment-result-data.ts

```diff
@@ -70,6 +70,7 @@ export interface TestStepResult {
 export interface AssessmentNavState {
     selectedTestSubview: RequirementName | GettingStarted;
     selectedTestType: VisualizationType;
+    expandedTestType?: VisualizationType;
```

Comment:
Why is this optional?

Reply:
It can be null or undefined if nothing is currently expanded